packages/mobile/src/screens/user-list-screen/UserList.tsx

```diff
@@ -47,16 +59,26 @@ export const UserList = (props: UserListProps) => {
     [usersMap, userIds]
   )
 
+  useEffect(() => {
+    if (users.length !== 0) {
+      cachedUsers.current = users
```

Comment:
this cache doesn't persist b/w "types" of user list screens right? I think it might. If you go from track => reposters => track => favorites

Reply:
i don't think so cause all each screen renders their own `UserList`!

Reply:
what if you go to a different track's userlist modal for reposters?